src/Controller/EnMarche/MembershipController.php

```diff
@@ -20,16 +20,28 @@
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Security\Core\Authorization\AuthorizationChecker;
 
 class MembershipController extends Controller
 {
+    /**
+     * Redirection for SEO
+     *
+     * @Route("/inscription", name="app_membership_redirect")
+     * @Method("GET|POST")
+     */
+    public function registerRedirectToAdhesionAction(Request $request): RedirectResponse
```

Comment:
Use `\AppBundle\Redirection\LegacyRedirectionsSubscriber` instead ;-)

Reply:
The query string part should be redirected too.

Reply:
ok